tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+  do_gsl_drag_tofd     = @[DO_GSL_DRAG_TOFD]
+  do_ugwp_v1_orog_only = @[DO_UGWP_V1_OROG_ONLY]
+  do_ugwp_v1_w_gsldrag = @[DO_UGWP_V1_W_GSLDRAG]
+  min_lakeice  = @[MIN_LAKEICE]
+  min_seaice   = @[MIN_SEAICE]
+  use_cice_alb = @[USE_CICE_ALB]
+  pdfcld       = .false.
+  fhswr        = @[FHSWR]
+  fhlwr        = @[FHLWR]
+  ialb         = @[IALB]
+  iems         = @[IEMS]
+  iaer         = @[IAER]
+  icliq_sw     = @[ICLIQ_SW]
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  elmx         = @[ELMX]
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = @[CNVGWD]
+  cpl_imp_mrg  = @[CPL_IMP_MRG]
+  shal_cnv     = @[SHAL_CNV]
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = @[HYBEDMF]
+  satmedmf     = @[SATMEDMF]
+  isatmedmf    = @[ISATMEDMF]
+  lheatstrg    = @[LHEATSTRG]
+  lseaspray    = @[LSEASPRAY]
+  random_clds  = @[RANDOM_CLDS]
+  trans_trac   = .true.
+  cnvcld       = @[CNVCLD]
+  imfshalcnv   = @[IMFSHALCNV]
+  imfdeepcnv   = @[IMFDEEPCNV]
+  do_deep      = @[DO_DEEP]
+  progsigma    = @[PROGSIGMA]
+  betascu      = @[BETASCU]
+  betamcu      = @[BETAMCU]
+  betadcu      = @[BETADCU]
+  betal_deep   = @[BETAL_DEEP]
+  betas_deep   = @[BETAS_DEEP]
+  ras          = @[RAS]
+  cdmbgwd      = @[CDMBWD_NEST02]
+  clam_deep    = @[CLAM_DEEP]
+  prslrd0      = 0.
+  ivegsrc      = 1
+  isot         = 1
+  lsoil        = 4
+  lsm          = @[LSM]
+  iopt_dveg    = @[IOPT_DVEG]
+  iopt_crs     = @[IOPT_CRS]
+  iopt_btr     = 1
+  iopt_run     = 1
+  iopt_sfc     = @[IOPT_SFC]
+  iopt_frz     = 1
+  iopt_inf     = 1
+  iopt_rad     = @[IOPT_RAD]
+  iopt_alb     = @[IOPT_ALB]
+  iopt_snf     = 4
+  iopt_tbot    = 2
+  iopt_stc     = @[IOPT_STC]
+  iopt_trs     = @[IOPT_TRS]
+  iopt_diag    = @[IOPT_DIAG]
+  sfc_rlm      = @[SFC_RLM]
+  sfc_z0_type  = @[SFC_Z0_TYPE]
+  debug        = .false.
+  nstf_name    = @[NSTF_NAME]
+  nst_anl      = .true.
+  psautco      = 0.0008,0.0005
+  prautco      = 0.00015,0.00015
+  lgfdlmprad   = @[LGFDLMPRAD]
+  effr_in      = @[EFFR_IN]
+  ldiag_ugwp   = @[LDIAG_UGWP]
+  do_RRTMGP          = @[DO_RRTMGP]
+  active_gases       = 'h2o_co2_o3_n2o_ch4_o2'
+  ngases             = @[NGASES]
+  lw_file_gas        = 'rrtmgp-data-lw-g128-210809.nc'
+  lw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-lw.nc'
+  sw_file_gas        = 'rrtmgp-data-sw-g112-210809.nc'
+  sw_file_clouds     = 'rrtmgp-cloud-optics-coeffs-sw.nc'
+  rrtmgp_nGptsSW     = 112
+  rrtmgp_nGptsLW     = 128
+  rrtmgp_nBandsLW    = 16
+  rrtmgp_nBandsSW    = 14
+  doGP_cldoptics_LUT = @[DOGP_CLDOPTICS_LUT]
+  doGP_lwscat        = @[DOGP_LWSCAT]
+  do_sppt      = @[DO_SPPT]
+  do_shum      = @[DO_SHUM]
+  do_skeb      = @[DO_SKEB]
+  frac_grid    = @[FRAC_GRID]
+  cplchm       = @[CPLCHM]
+  cplflx       = @[CPLFLX]
+  cplice       = @[CPLICE]
+  cplwav       = @[CPLWAV]
+  cplwav2atm   = @[CPLWAV2ATM]
+  lndp_type    = @[LNDP_TYPE]
```

Comment:
The current global_workflow does not have `lndp_type`

Reply:
lndp_type is set to the namelist default, so I should be able to remove this one